### PR TITLE
test: pin and unpin message test added

### PR DESCRIPTION
**Pull request by Valentina1133**
CI  run: https://ci.status.im/job/status-desktop/job/e2e/job/manual/1294/
Allure:
Screenshot 2024-01-24 at 21 09 23

### CODE REVIEW

**Review by saledjenic — APPROVED** [review]

File: tests/communities/test_communities_pin_and_unpin_messages.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    with step('Verify that the message was pinned'):
+        assert message.message_is_pinned
+        assert message.pinned_info_text + message.user_name_in_pinned_message == 'Pinned by' + user_account.name
+        assert message.get_message_color() == ColorCodes.ORANGE.value
+
+    with step('Hover message and unpin it'):
+        message.hover_message().toggle_pin()
+
+    with step('Verify that the message was pinned'):
```
Comment (anastasiyaig) [inline tests/communities/test_communities_pin_and_unpin_messages.py:59]:
```suggestion
    with step('Verify that the message was unpinned'):
```

File: tests/communities/test_communities_pin_and_unpin_messages.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        assert not message.message_is_pinned
+        assert message.user_name_in_pinned_message == ''
+        assert not message.get_message_color() == ColorCodes.ORANGE.value
```
Comment (anastasiyaig) [inline tests/communities/test_communities_pin_and_unpin_messages.py:62]: What if we instead verify that there is no pinned messages section available at the top (if we unpin the very last message)?


https://github.com/status-im/desktop-qa-automation/assets/82375995/c3f76fbb-6608-498d-8129-9c1c00e5712e

**Review by anastasiyaig — APPROVED** [review]

File: tests/communities/test_communities_pin_and_unpin_messages.py (hunk 2 above)
Comment (Valentina1133) [reply tests/communities/test_communities_pin_and_unpin_messages.py:62]: Yes, good idea, I will add it

File: tests/communities/test_communities_pin_and_unpin_messages.py (hunk 1 above)
Comment (Valentina1133) [reply tests/communities/test_communities_pin_and_unpin_messages.py]: Fixed

File: tests/communities/test_communities_pin_and_unpin_messages.py (hunk 2 above)
Comment (Valentina1133) [reply tests/communities/test_communities_pin_and_unpin_messages.py]: Done

**Pull request merged**